scripts/core/editor3/reducers/tests/suggestions_style.spec.jsx

```diff
@@ -53,7 +53,7 @@ describe('editor3.reducers.suggestion.CREATE_CHANGE_STYLE_SUGGESTION', () => {
         });
     });
 
-    it('should set BOLD suggestion for a bold text, the BOLD style not present anymore for selected text', () => {
+    it('should reset BOLD type when set BOLD suggestion for a bold text', () => {
```

Comment:
This might be more clear

It should create `TOGGLE_BOLD_SUGGESTION` when toggling BOLD on text which was already bold before enabling suggestion mode